src/components/Logo/Logo.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import styles from './Logo.scss';
+import img from '../../assets/pictures/logo.png'
+
+export default function Logo() {
+  return (
+    <span>
+      <Link to="/"><img src={img} width="240px" height="80px" className = {styles.logo}/></Link>
```

Comment:
Please remove the space between `className` and `=`

```suggestion
      <Link to="/"><img src={img} width="240px" height="80px" className={styles.logo}/></Link>
```